Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -700,7 +700,7 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
             }
 
             Logger.networkProtection.log("🔴 Stopping VPN due to no auth token")
-            await attemptShutdownDueToRevokedAccess()
+            await cancelTunnel(with: TunnelError.startingTunnelWithoutAuthToken)
```

Comment:
This change will result in the VPN retrying repeatedly until success, but we have a 15 second delay between attempts so this is intended; it will allow the VPN to finish reconnection after the Keychain is available.